src/components/draft/CubeDraft.tsx

```diff
@@ -201,7 +258,7 @@ const CubeDraft: React.FC<CubeDraftProps> = ({ draft, socket }) => {
         const dragTime = Date.now() - (dragStartTime ?? 0);
 
         if (dragTime < 200) {
-          return onClickCard(source.index);
+          return selectCardByIndex(source.index);
```

Comment:
selectCardByIndex determines the card in the pack from its index, then calls applyCardSelectionForStep. Here we do that because the drag and drop handler has detected a click.
Below when the handler determines it is a drag, we also call applyCardSelectionForStep if the source of the drag was the pack and the target the deck or sideboard (the latter cannot occur given the sideboard deck stack is still commented out).

On any other drag we do moveCardBetweenDeckStacks which handles moving cards within a deck stack or across them (which again cannot occur with the sideboard commented out). Drags with the pack as the target do nothing since that is cleraly illogical (unless conspiracy draft rules could be implemented lol)